src/pages/Lending/Pool/components/Repay/RepayInput.tsx

```diff
@@ -341,23 +455,24 @@ export const RepayInput = ({
   }, [_isSmartContractAddress])
 
   const quoteErrorTranslation = useMemo(() => {
-    if (!isThorchainLendingRepayEnabled) return 'lending.errors.repaymentsDisabled'
-    if (_isSmartContractAddress) return 'trade.errors.smartContractWalletNotSupported'
-    if (!hasEnoughBalanceForTxPlusFees || !hasEnoughBalanceForTx) return 'common.insufficientFunds'
+    if (!isThorchainLendingRepayEnabled) return translate('lending.errors.repaymentsDisabled')
+    if (_isSmartContractAddress) return translate('trade.errors.smartContractWalletNotSupported')
+    if (!hasEnoughBalanceForTxPlusFees || !hasEnoughBalanceForTx)
+      return translate('common.insufficientFunds')
     if (isLendingQuoteCloseError) {
       if (
         /not enough fee/i.test(lendingQuoteCloseError.message) ||
         /not enough to pay transaction fee/i.test(lendingQuoteCloseError.message)
       )
-        return 'trade.errors.amountTooSmallUnknownMinimum'
+        return translate('trade.errors.amountTooSmallUnknownMinimum')
       if (
         /loan hasn't reached maturity/i.test(lendingQuoteCloseError.message) ||
         /loan repayment is unavailable/i.test(lendingQuoteCloseError.message)
       )
-        return 'Repayment not yet available'
+        return translate('Repayment not yet available')
```

Comment:
need to add a translation for this if we don't have one

Reply:
9f472ee426